### ISSUE TITLE
[BUG] Market name DDEX is invalid

### ISSUE THREAD

**Issue by znerol19619** [CoinAlpha/hummingbot#1233]
**Describe the bug**
Error during initialization, after ``start`` command bot will throw an error message that ddex is not a valid market name.

**Steps To Reproduce**
1. Setup a configuration with PMM/XEMM/ARB
2. Choose DDEX as the exchange
2. Run ``start`` command
...

**Screenshots**
![pmm_ddexerror](https://user-images.githubusercontent.com/27669951/71320581-1d407e80-24e8-11ea-92bd-af4a7cdb2898.png)
![xemm_ddexerror](https://user-images.githubusercontent.com/27669951/71320591-36e1c600-24e8-11ea-8a05-436ab1c02128.png)

**Error**
```
2019-12-22 09:47:24,026 - 10 - root - ERROR - Unhandled error in background task: Market name ddex is invalid.
Traceback (most recent call last):
  File "/hummingbot/core/utils/async_utils.py", line 8, in safe_wrapper
    return await c
  File "/hummingbot/client/command/start_command.py", line 86, in start_market_making
    start_strategy(self)
  File "/hummingbot/strategy/cross_exchange_market_making/start.py", line 51, in start
    self._initialize_markets(market_names)
  File "/hummingbot/client/hummingbot_application.py", line 364, in _initialize_markets
    raise ValueError(f"Market name {market_name} is invalid.")
ValueError: Market name ddex is invalid.
```
**Release version**
```
                "branch": "development",
                "commit": "5a94dbb9d8876dd0ab2f5d166dd90193e0a25e72",
                "date": "2019-12-21T00:32:40Z",

```

**Attachments**
[conf-and-logs.zip](https://github.com/CoinAlpha/hummingbot/files/3992338/conf-and-logs.zip)

**Comment by BUDDHA-TONY** [comment]
Also tested on Discovery Strategy 

![image](https://user-images.githubusercontent.com/55869559/71331150-cb413c80-256b-11ea-9348-17618c472c06.png)

**Comment by BUDDHA-TONY** [comment]
Market name DDEX is invalid - FIX / #1240 merged in to development

PMM - success
XEMM - success / paper-trade off due to known issue #1232
ARB - success
DISCOVERY - success

**Comment by znerol19619** [comment]
thanks @BUDDHA-TONY , I will now close this bug report.
